lib/run-status.js

```diff
@@ -134,9 +135,10 @@ RunStatus.prototype.prefixTitle = function (file) {
 	var separator = ' ' + chalk.gray.dim(figures.pointerSmall) + ' ';
 
 	var prefix = path.relative('.', file)
-		.replace(this.base, '')
+		.replace(new RegExp('^' + escapeStringRegexp(this.base)), '')
```

Comment:
I like the last one! Perhaps add a comment "Only replace base if it's at the start"?